### ISSUE TITLE
Bring TK label translations into template

### ISSUE THREAD

**Issue by steve-taylor-wsu** [MukurtuCMS/Mukurtu-CMS#79]
We need to bring TK label translations into the TK label template. We aren't really concerned with display at this point, we only need the data available in the template so it's accessible when we get to front-end.

Things that might be useful:
* Add template variable in `mukurtu_local_contexts.module` in `local_contexts_label`
* Populate and pass that variable in `LocalContextsLabelFormatter.php`
* In the template we'll need access to the translation locale, language, name, and text
* Extending or adding queries in `LocalContextsLabel.php`
* Database schema for all the TK Label stuff including translations is in `mukurtu_local_contexts.install`. This data will already exist if you add supported Local Contexts projects to your site
* Here are some project IDs you can test with (there are a couple translations buried in here...):
  * 92a31c10-eccf-40a8-a173-ac7be9e8e5ab
  * bfc3dcf2-a78d-4fec-9fe1-accc6c675b04
  * 687a772e-5ca0-4235-b5c0-d88fe548745b

Also feel free to review/suggest changes to any of the LocalContexts* classes. It's all my first effort and I have some inconsistences; it's not set in stone.

**Comment by steve-taylor-wsu** [comment]
Also don't forget you can do a `ddev describe` to get to phpMyAdmin to look at the database.

**Comment by steve-taylor-wsu** [comment]
@nick-deer I didn't think of this before, but I think it'd make more sense if we indexed the translations in the templates by locale.

Rather than
```
[0 => [ "locale" => "es", ...]]
```

do
```
["es" => [ "locale" => "es", ...]]
```

Two problems to work around here:
1. The LC Hub lets you leave locale blank or as whitespace. In this case my first instinct is to default to incremental numeric keys, but I'm open to suggestions.
2. I'm fairly certain the LC Hub lets you specify multiple translations with the same locale. I don't think this is intentional, I just don't think they've thought far enough to enforce any rules here. My first instinct here is to disambiguate with an incremental delta like Drupal so loves to do, e.g., keys 'es', 'es-1', 'es-2'. The other option would be to move to a nested array like 
```
"es" => [["text" => "translation 1"...], ["text" => "translation 2"...]]
```
Thoughts on this? I personally think people altering themes/templates would dislike nested arrays (certainly if we mixed flat & nested).
